CollAction/Helpers/ProjectNormalizeStringExtensions.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Text;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CollAction.Helpers
+{
+    public static class ProjectNormalizeStringExtensions
+    {
+
+        public static string ToUrlSlug(this string value){
+
+            //First to lower case
+            value = value.ToLowerInvariant();
+
+            //Replace spaces
+            value = Regex.Replace(value, @"\s", "-", RegexOptions.Compiled);
```

Comment:
I think it's probably best if you make your Regex static if you use compiled, otherwise it'll probably just take longer.